packages/react/src/themes/teams/components/embed/embedStyles.ts

```diff
@@ -0,0 +1,57 @@
+import { ComponentSlotStylesInput, ICSSInJSStyle } from '../../../types'
+import { pxToRem } from 'src/lib'
+import { EmbedProps, EmbedState } from '../../../../components/Embed/Embed'
+
+export default {
+  root: ({ props, variables }): ICSSInJSStyle => {
```

Comment:
```suggestion
  root: ({ props: p, variables: v }): ICSSInJSStyle => {
```

We decided to use `p` & `v` as it is more obvious in styles